src/uct/ib/base/ib_iface.c

```diff
@@ -380,8 +382,20 @@ static ucs_status_t uct_ib_iface_init_lmc(uct_ib_iface_t *iface,
     return UCS_OK;
 }
 
+static int uct_ib_find_lcs_cpu(ucs_cpu_set_t const *cpu_cores)
```

Comment:
only solution is moving leaving it 'as is' or moving it to cpu_set.c and adding it to Makefile